mlrun/run.py

```diff
@@ -389,6 +389,9 @@ def import_function_to_dict(url, secrets=None):
     code = get_in(runtime, "spec.build.functionSourceCode")
     update_in(runtime, "metadata.build.code_origin", url)
     cmd = code_file = get_in(runtime, "spec.command", "")
+    # use kind = "job" by default if not specified
+    if "kind" not in runtime:
+        runtime["kind"] = "job"
```

Comment:
for the CLI `build` command, this will only handle the flow where the (optional) `func_url` argument is passed.
What about the case where a dict is passed to the `--runtime` flag, and it doesn't contain a kind?
Add a unittest for this case as well